backend/fms_core/models/sample_property.py

```diff
@@ -0,0 +1,19 @@
+import reversion
+
+from .tracked_model import TrackedModel
+from django.db import models
+from .sample import Sample
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+from ._validators import name_validator
+
+
+@reversion.register()
+class SampleProperty(TrackedModel):
+    """ Class to store additional sample metadata. """
+
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, validators=[name_validator],
+                            help_text="The name of the property.")
+    value = models.CharField(max_length=200, help_text="The value of the property.s")
```

Comment:
I don't think there is a cost to longer max_length in the DB ... The fields are varchar and are allocated the length required. Since this field could contain anything i would recommend giving it longer than 200 ... maybe 2000 ? ... this is not the allocated size as far as i know only the max permitted by us.